client/bootstrap_demo_community.py

```diff
@@ -285,7 +285,7 @@ def main(ipfs_local, client, url, port, spec_file, test):
     client.await_block(1)
 
     if(not test):
-        print(f"Community {cid} sucessfullly bootatrapped")
+        print(f"Community {cid} successfully bootstrapped")
```

Comment:
This is the only line I read when running the script. I could no longer bear it XD